tests/test-sources/plugins/languages/rust/rustaceanvim.nix

```diff
@@ -115,7 +115,7 @@
               return { 'rust-analyzer', '--log-file', RustaceanConfig.server.logfile }
             end
           '';
-          settings = ''
```

Comment:
Should this be `default_settings.rust-analyzer` as it's a function?

Reply:
No, the attrset is under rust-analyzer

Reply:
You're right, I read the note in the warning backwards 😅